### ISSUE TITLE
Lolin/Wemos S2-Pico has build in display but not defined

### ISSUE THREAD

**Issue by dglaude** [adafruit/circuitpython#7325]
### CircuitPython version

```python
Adafruit CircuitPython 8.0.0-beta.5 on 2022-12-08; S2Pico with ESP32S2-S2FN4R2
Board ID:lolin_s2_pico
```

### Code/REPL

```python
import board
dir(board)
```

### Behavior

```python
['__class__', '__name__', 'BUTTON', 'I2C', 'IO0', 'IO1', 'IO10', 'IO11', 'IO12', 'IO13', 'IO14', 'IO15', 'IO16', 'IO17', 'IO18', 'IO2', 'IO21', 'IO3', 'IO33', 'IO34', 'IO35', 'IO36', 'IO37', 'IO38', 'IO4', 'IO5', 'IO6', 'IO7', 'IO8', 'IO9', 'LCD_RST', 'LED', 'MISO', 'MOSI', 'SCK', 'SCL', 'SDA', 'SS', 'board_id']
```

### Description

That board has a build in display (see https://circuitpython.org/board/lolin_s2_pico/) connected in I2C:
```128 x 32 SSD1306 OLED display internally connected to the same I2C bus as the external port, reset pin connected to GPIO18, I2C address 0x3C``` ( from https://circuitpython.org/board/lolin_s2_pico/ ).

But DISPLAY is not defined in the pin (problem 1).

(problem 2) maybe as a consequence, the screen does not display the REPL or Wifi information.

Anyway, with 32x128 you cannot see a lot.

So I wonder if the normal expected behavior for such a board is for CP to initialize the screen and use it.

And if the solution for problem 1 and problem 2 are different, maybe it involve adding a build in module or more.

### Additional information

The screen is working fine, you just have to initialize it:
```python
displayio.release_displays()
i2c = board.I2C()  # uses board.SCL and board.SDA
oled_reset=board.LCD_RST
display_bus = displayio.I2CDisplay(i2c, device_address=0x3C, reset=oled_reset)
display = adafruit_displayio_ssd1306.SSD1306(display_bus, width=128, height=32)
```

**Comment by evildave666** [comment]
This is the only board with Circuitpython support with an I2C built-in display, hence there is no example code to crib from. At some point I was going to look into setting it up but lacked the time.

**Comment by dglaude** [comment]
Great thanks, I started to search for an example... next step was to ask on discord.
I think the situation is fine as it is, maybe we can update circuitpython.org to do expectation management?

PS: I acquired one specimen of each of the LOLIN boards with circuitpython support, just to try, so thank you for your work.

**Comment by dglaude** [comment]
Fixed by #7754
